### ISSUE TITLE
Single Pipeline

### ISSUE THREAD

**Issue by rhiananthony** [HumanCellAtlas/dcp#72]
As users of the DCP, I expect to see at least one pipeline producing analysis results in the DCP when it reaches it's pilot phase.

**Comment by rhiananthony** [comment]
Comments from @kbergin in original ticket in trello:

2018-07-10 10:30PM
Pipeline is approved, we just need to finish standing up prod with it!

2018-04-17 11:11 AM
We are meeting with the new subgroup of the AWG tomorrow and hopefully will get more information on how to get this.

2018-03-08 12:06 PM
Blocked by decision from governance on whether this is ok/is blessed

**Comment by kbergin** [comment]
Closing as this is represented in another ticket
